api-gateway/src/main/java/com/petclinic/bffapigateway/dtos/Auth/Register.java

```diff
@@ -24,6 +25,7 @@ public class Register {
     private String userId;
     private String email;
     private String username;
+    private final String defaultRole = Roles.OWNER.toString();
```

Comment:
Why are you adding the roles in the DTO's instead of specifying them in the logic?